src/app/about/page.tsx

```diff
@@ -1,19 +1,49 @@
+import React from "react";
 import Image from "next/image";
 import styles from "./About.module.css";
 
 import OfficerCard from "./officer-card";
-import officerCardData from "./officer-data";
+import { positionData } from "./officer-data";
 
 export default function About() {
-  const officerCards = officerCardData.map((cardData) => (
-    <OfficerCard
-      key={cardData.position}
-      position={cardData.position}
-      image={cardData.image}
-      name={cardData.name}
-      socialLinksData={cardData.socialLinksData}
-    />
-  ));
+  // const officerCards = officerCardData.map((cardData) => (
+  //   <OfficerCard
+  //     key={cardData.position}
+  //     position={cardData.position}
+  //     image={cardData.image}
+  //     name={cardData.name}
+  //     socialLinksData={cardData.socialLinksData}
+  //   />
+  // ));
+
+  const officerCardSection = Object.entries(positionData).map(
+    ([semester, officers]) => {
+      const officerCards = Object.entries(officers).map(
+        ([position, officer]) => {
+          if (officer === undefined) {
+            return <div key={`${semester} ${position}`} />;
+          }
+
+          return (
+            <OfficerCard
+              key={`${semester} ${position}`}
```

Comment:
[Please Change]: We can use just position as the key here because it's unique within each semester. In other words; we don't need to ensure the keys are unique across both loops; just the current one.